pages/docs/tutorials/kafka/configure-kafka-avro.md

```diff
@@ -0,0 +1,162 @@
+
+While AsyncAPI schemas can be the default choice for describing payloads, many prefer using Avro Schemas to define messages in Kafka. Through this tutorial, you'll learn to modify your existing AsyncAPI schema to add Avro schema into your document in both YAML and JSON formats.
+
+## Background Context
+
+[AsyncAPI](https://www.asyncapi.com/) is a specification for describing Event-Driven Architectures (EDAs) in a machine-readable format. It allows developers to define how messages are exchanged between different services by providing an agreement for communication.
+
+[Avro](https://avro.apache.org/) is an efficient binary serialization format, used to ensure schema-aware communications between the messages within Apache Kafka. With Avro Schema there is a standardized method for serializing data allowing for interoperability and schema evolution capabilities. Avro promotes data exchange among systems by offering a common schema that fosters compatibility, between different components. 
+
+## Defining message payload with Avro Schema directly in the AsyncAPI document
+
+Defining message schemas with the default schema is already covered in the [previous tutorial](https://deploy-preview-2252--asyncapi-website.netlify.app/docs/tutorials/kafka). The default choice was the AsyncAPI schemas which is a superset of JSON Schema that looked as follows, 
+
+```
+messages:
+  userSignedUp:
+    payload:
+      type: object
+      properties:
+        user-id:
+          type: integer
+          description: This property describes the id of the user
+        user-email:
+          type: string
+          description: This property describes the id of the user
+```
+
+In this section, let's shift our focus to defining messages using Avro Schemas directly within our document. 
+
+```
+messages:
+  userSignedUp:
+    payload:
+      schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
+      schema:
+        type: record
+        name: UserSignedUp
+        namespace: com.company
+        doc: User sign-up information
+        fields:
+          - name: user-id
+            type: int
+          - name: user-email
+            type: string
+```
+
+In the above snippet: 
+- The `userSignedUp` message is defined with Avro Schema, using the specified `schemaFormat` and the `schema`.
+- Use the `schemaFormat` to specify using a MIME type, that you use Avro and what version of Avro Schema.
+- The `schema` includes a `record` named `UserSignedUp` within the `com.company` namespace. It also describes two fields, `user-id` and `user-email`, defining their data types as `int` and `string` respectively.
+
+By combining the Avro Schema discussed above into the previous tutorial, you'll have an AsyncAPI document fully equipped with Avro Schema!
+
+```
```

Comment:
please always test documents you share in docs with studio.asyncapi.com to make sure they actually work

this is valid document

```
asyncapi: 3.0.0
info:
  title: User Signup API
  version: 1.0.0
  description: The API notifies you whenever a new user signs up in the application.
servers:
  kafkaServer:
    host: test.mykafkacluster.org:8092
    description: Kafka Server
    protocol: kafka
operations:
  onUserSignedUp:
    action: receive
    channel:
      $ref: '#/channels/userSignedUp'
channels:
  userSignedUp:
    description: This channel contains a message per each user who signs up in our application.
    address: user_signedup
    messages:
      userSignedUp:
        $ref: '#/components/messages/userSignedUp'
components:
  messages:
    userSignedUp:
      payload:
        schemaFormat: 'application/vnd.apache.avro;version=1.9.0'
        schema:
          type: record
          name: UserSignedUp
          namespace: com.company
          doc: User sign-up information
          fields:
            - name: userId
              type: int
            - name: userEmail
              type: string
```

please also look at https://asyncapi.slack.com/archives/CQVJXFNQL/p1704890017575799

so in this and probably best also in the other PR, update `user-id` and user-email` to `userId` and `userEmail` - even in AsyncAPI schema to be consistent and not cause confusion